kia-lib/src/main/kotlin/com/faforever/ice/ice4j/AgentWrapper.kt

```diff
@@ -103,14 +104,21 @@ class AgentWrapper(
                     setState(IceState.AWAITING_CANDIDATES)
                 }
 
+                var allowHost = true
+                var allowReflexive = true
+                if (forceRelay) {
+                    allowHost = false
+                    allowReflexive = false
+                }
```

Comment:
In Kotlin avoid `var` if possible. if/else is an expression, so you could say
`val bla = if(somethinge) 1 else -1`

In this case `val allowHost = !forceRelay` and `val allowReflex = !forceRelay` seems better though.

Reply:
Or you pass `!forceRelay` directly as argument in the method call